src/hooks/use-like-card.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { LocalStorageService } from '../services/local-storage-service';
+
+const CARDS_IDS = 'likedCards'
+
+const useLikeCard = () => {
```

Comment:
Vienai kortelei naudojate visų kortelių duomenis,
šis hooksas turėtų būti naudojamas kortelių "wrapper"io komponente, ir kikvienai koretelei perduodami tik tai kortelei aktualūs duomenys